platforms/desktop/windows.md

```diff
@@ -1 +1,54 @@
-# Langley Foxall Windows Apps
\ No newline at end of file
+# Langley Foxall Windows Apps
+
+Whilst our windows application development is slightly more rare than the other platforms, we still have a set standard 
+of development, built by our in-house specialists.
+
+As a preference, our Windows applications are developed using C#.
+
+### Universal Windows Platform (UWP)
+
+The Universal Windows Platform is our windows platform of choice, allowing you to develop and build once for Desktop, 
+Mobile, Xbox, Hololens and more.
+
+We work mainly with C# for our logic layer, and build our views using XAML.
+
+#### Things you should know
+
+- The UWP Platform is only compatible on Windows 10, with Windows 8 having a similar framework called Windows RT.
+- The UWP & WinRT platforms use .NET Core - as opposed to the full .NET Framework. For Windows 10 this has recently
+extended to .NET Core 2.0, which added a lot of extra API's.
+
+#### Design Patterns / Architecture
+
+UWP favours the MVVM design pattern (Model, View, View-Model), for separation of the views, business logic and models.
```

Comment:
Might be worth linking the MVVM design pattern to a reference [such as a wiki page](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel)